### ISSUE TITLE
How to overwrite sparklyr data frame to the file format???

### ISSUE THREAD

**Issue by Venkatraman92** [rstudio/sparklyr#569]

**Comment by edgararuiz** [comment]
Hi @Venkatraman92 , can you give me some more background of what you're trying to accomplish?

**Comment by Venkatraman92** [comment]
Hi @edgararuiz , 
Once i have written .csv file using Spark_write_csv function from Spark data frame it got successful but again I tried to write .csv with same name it is show an error mentioned bellow, while writing csv have mentioned Overwrite so that it can replace previous one but it's not working.

final_data <- spark_write_csv(sparkdataframe,"/myfolder/data.csv",header=TRUE,null_value=NULL,overwrite=TRUE)

error:
unused argument(overwrite=TRUE)

**Comment by Venkatraman92** [comment]
Hi @javierluraschi  
Thanks for the help. Its working fine.

**Comment by chezou** [comment]
@Venkatraman92 it seems almost fine about this issue via #579 . Why don't you close this issue?

**Comment by javierluraschi** [comment]
@Venkatraman92 Implemented in development `sparklyr` and planned to be released in `sparklyr 0.6`. With latest version, try:

```
spark_write_csv(tbl, "s3://bucket/dir/", mode = "overwrite")
```

See: https://github.com/rstudio/sparklyr/pull/579
